File: Administrativo/spring-erp-web/SpringNg/src/app/erp_module/shared/dominio/dto/DtoTabla.ts
import { ParametroPaginacionGenerico } from '../ParametroPaginacionGenerico';
import { DomSanitizer } from '@angular/platform-browser';

export class DtoTabla {
    id: number;
    codigo: string;
    codigoNumerico: number;
    nombre: string;
    descripcion: string;
    estado: string;
    value: any;
    label: string;
    internalValue: any;
    valorFlag: string;
    valorNumerico: number;
    secuencia: number;
    idTipoAyuda: string;
    idResultado: string;
    idSubCondicion: string;
    base64: string;
    valor1: string;
    valor2: string;
    valor3: string;
    explicacion: string;
    porcentaje: number;
    paginacion: ParametroPaginacionGenerico;
    min: number;
    max: number;
    color: string;

    num1: number;
    num2: number;
    num3: number;
    num4: number;
    num5: number;
    num6: number;
    num7: number;
    num8: number;
    num9: number;

    fechainicio1: Date;
    fechafin1:Date;
    fechainicio2: Date;
    fechafin2:Date;

    constructor() {
        this.paginacion = new ParametroPaginacionGenerico();
    }
}
export class DtoPublicacionVista {

    private _inner: any = "";
    tamanio: string = "";
    altura: number = 100;

    constructor(private sanitizer: DomSanitizer) {

    }
    get inner(): any {
        return this.sanitizer.bypassSecurityTrustHtml(this._inner);
    }
    set inner(theBar: any) {
        this._inner = theBar;
    }
}

File: Administrativo/spring-framework-util/net/royal/spring/framework/core/dominio/dto/DtoTabla.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace net.royal.spring.framework.core.dominio.dto {
    public class DtoTabla {
        private String _Codigo;

        public String Codigo {
            get { return (UString.estaVacio(_Codigo)) ? null : _Codigo.Trim(); }
            set { _Codigo = value; }
        }



        public DtoTabla() {
            paginacion = new ParametroPaginacionGenerico();
        }

        public DtoTabla(String Codigo, String Nombre) {
            this.Codigo = Codigo;
            this.Nombre = Nombre;
        }

        public DtoTabla(Nullable<Int32> CodigoNumerico) {
            this.CodigoNumerico = CodigoNumerico;
        }

        public Nullable<Int32> CodigoNumerico { get; set; }
        public String Nombre { get; set; }
        public String Estado { get; set; }
        public String Descripcion { get; set; }
        public Int32? Id { get; set; }
        public int? value { get; set; }
        public Nullable<Int32> Secuencia { get; set; }
        public string label { get; set; }
        public bool internalValue { get; set; }
        public String base64 { get; set; }

        public String Explicacion { get; set; }
        public Nullable<Decimal> Porcentaje { get; set; }
        public Nullable<Decimal> valorNumerico { get; set; }
        public String valorFlag { get; set; }

        public String IdTipoAyuda { get; set; }
        public String IdResultado { get; set; }
        public Double? IdResultadoNumerico { get; set; }

        public String IdSubCondicion { get; set; }

        public String valor1 { get; set; }
        public String valor2 { get; set; }
        public String valor3 { get; set; }

        public Int32? num1 { get; set; }
        public Int32? num2 { get; set; }
        public Int32? num3 { get; set; }

        public Int32? num4 { get; set; }
        public Int32? num5 { get; set; }
        public Int32? num6 { get; set; }

        public Int32? num7 { get; set; }
        public Int32? num8 { get; set; }
        public Int32? num9 { get; set; }

        public String dato1 { get; set; }
        public String dato2 { get; set; }

        public Nullable<DateTime> fechainicio1 { get; set; }
        public Nullable<DateTime> fechafin1 { get; set; }

        public Nullable<DateTime> fechainicio2 { get; set; }
        public Nullable<DateTime> fechafin2 { get; set; }


        public ParametroPaginacionGenerico paginacion { get; set; }
        public static List<DtoTabla> obtenerLista(String[] listaCodigos) {
            List<DtoTabla> lst = new List<DtoTabla>();
            if (listaCodigos == null)
                return lst;
            for (int i = 0; i < listaCodigos.Length; i++) {
                if (!UString.estaVacio(listaCodigos[i])) {
                    lst.Add(new DtoTabla(listaCodigos[i], null));
                }

            }
            return lst;
        }

    }
}
